dogstatsd-client/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021-Present Datadog, Inc. https://www.datadoghq.com/
+// Copyright 2024-Present Datadog, Inc. https://www.datadoghq.com/
```

Comment:
I have no clue what the correct answer is, but should we be modifying the license headers like this? I'm not sure what is legally correct.

Reply:
iirc it doesn't strictly matter as long as the header is there when the file is created